fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -233,69 +237,60 @@ protected RdfStream getResourceTriples(final int limit) {
 
         final LdpPreferTag ldpPreferences = new LdpPreferTag(returnPreference);
 
-        final RdfStream rdfStream = new RdfStream();
-
         final Predicate<Triple> tripleFilter = ldpPreferences.prefersServerManaged() ? x -> true :
             IS_MANAGED_TRIPLE.negate();
 
-        if (ldpPreferences.prefersServerManaged()) {
-            rdfStream.concat(getTriples(LdpRdfContext.class));
-        }
-
-        rdfStream.concat(filter(getTriples(TypeRdfContext.class), tripleFilter::test));
+        final List<Stream<Triple>> streams = new ArrayList<>();
 
-        rdfStream.concat(filter(getTriples(PropertiesRdfContext.class), tripleFilter::test));
+        streams.add(getTriples(PROPERTIES).filter(tripleFilter));
 
-        if (!returnPreference.getValue().equals("minimal")) {
+        if (returnPreference.getValue().equals("minimal")) {
+            if (ldpPreferences.prefersServerManaged()) {
+                streams.add(getTriples(SERVER_MANAGED)
```

Comment:
@ajs6f we could certainly pull apart this PR and do it more incrementally, but in all likelihood, I wouldn't have time to do that until April, which probably means completely starting over (which means probably not doing it at all). Unless someone else wants to take over the shepherding of this PR.